modules/clickhouse/clickhouse.go

```diff
@@ -72,6 +79,53 @@ func (c *ClickHouseContainer) ConnectionString(ctx context.Context, args ...stri
 	return connectionString, nil
 }
 
+// ZookeeperOptions arguments for zookeeper in clickhouse
+type ZookeeperOptions struct {
+	Host, Port string
+}
+
+// renderZookeeperConfig generate default zookeeper configuration for clickhouse
+func renderZookeeperConfig(settings ZookeeperOptions) ([]byte, error) {
+	tpl, err := template.New("bootstrap.yaml").Parse(zookeeperConfigTpl)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse zookeeper config file template: %w", err)
+	}
+
+	var bootstrapConfig bytes.Buffer
+	if err := tpl.Execute(&bootstrapConfig, settings); err != nil {
+		return nil, fmt.Errorf("failed to render zookeeper bootstrap config template: %w", err)
+	}
+
+	return bootstrapConfig.Bytes(), nil
+}
+
+// WithZookeeper pass a config to connect clickhouse with zookeeper and make clickhouse as cluster
+func WithZookeeper(host, port string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
+		f, err := os.CreateTemp("", "clickhouse-tc-config-")
+		if err != nil {
+			panic(err)
```

Comment:
I think we should document that this function could panic under this condition. Would you mind adding it to the docs?

Reply:
I went ahead and added the docs 🙏  Once the CI passes, I think we can merge this one. Thanks!